db/migrate/20170501201633_add_canceled_to_order_detail.rb

```diff
@@ -4,10 +4,11 @@ def up
     add_column :order_details, :canceled_by, :integer
     add_column :order_details, :canceled_reason, :string
 
-    Reservation.where.not(order_detail_id: nil).find_each do |reservation|
-      reservation.order_detail.update_attributes(canceled_at: reservation.canceled_at,
+    Reservation.where.not(order_detail_id: nil, canceled_at: nil).find_each do |reservation|
+      reservation.order_detail.assign_attributes(canceled_at: reservation.canceled_at,
```

Comment:
Looks like the `save(validate: false)` still runs callbacks. `update_columns` is definitely faster.

Also, adding `includes(:order_detail)` into the query is making this much faster as well.

I just ran it with the includes and `update_columns` and it definitely looks faster (I'm seeing UPDATEs fly by much faster).

Usually I don't care too much about the performance of migrations, but once they go over a couple minutes I start to get worried. If we were to want to really boost this, we'd do it in a single `UPDATE` clause, but that's not as obvious what's going on.